proposals/STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,23 @@
+*[This document is work in progress. It will be moved to https://github.com/kubeflow/community/STEERING-COMMITTEE.md upon completion and approval.]*
+
+The Kubeflow Steering Committee (KSC) is responsible for the general health of the Kubeflow community.
+
+## Interim KSC
+During the CNCF incubation period, KSC will be a small council that consists of the following people:
+| &nbsp;                                                         | Member                    | Organization | Profile                                              |
+| -------------------------------------------------------------- | ------------------------- | ------------ | ---------------------------------------------------- 
+| <img width="30px" src="https://github.com/theadactyl.png"> | Thea Lamkin          | Google       | [@theadactyl](https://github.com/theadactyl) |
+| <img width="30px" src="https://github.com/james-jwu.png">     | James Wu            | Google  | [@james-jwu](https://github.com/james-jwu)         |
+| <img width="30px" src="https://github.com/jbottum.png">          | Josh Bottum                | Kubeflow      | [@jbottum](https://github.com/jbottum)                   |
+
+The responsibilities of the Interim KSC includes:
+1. Establish KSC, KCC and KTOC
+1. Create committee charters
+1. Work with CNCF TOC to complete the due diligence review
+
+## Kubeflow Steering Committee
+The governance of Kubeflow is an open, living process and document, and will continue to evolve as the community and project change.
+
+The Interim KSC is chartered to establish KSC. The interim committee will propose details of the KSC charter, roles, processes and limitations under this document.
+
+KSC will replace the interim committee after CNCF accepts Kubeflow as an incubation project.
```

Comment:
Could we specify that KSC will be formed based on the process defined by the interim KSC? As it reads, it sounds like KSC will be established by the interim KSC, but it's the process and the charter for which the interim KSC is responsible for, not the actual choosing of the members. If that's correct, I think additional clarification would be helpful. 

```suggestion
KSC will be formed based on the process defined by the interim KSC and will replace the interim committee after CNCF accepts Kubeflow as an incubation project.
```

Reply:
Sounds good.